docs-api/frontity-packages/features-packages/google-ad-manager.md

```diff
@@ -14,7 +14,7 @@ This package enables Frontity to integrate with Google Ad Manager. It allows you
     - [The `props` property](google-ad-manager.md#the-props-property)
 - [Examples](google-ad-manager.md#examples)
 - [Usage](google-ad-manager.md#usage)
-  - [Using the `Slot & Fill` pattern](google-ad-manager.md#using-the-slot-fill-pattern)
+  - [Using the `Slot & Fill` pattern](google-ad-manager.md#using-the-slot--fill-pattern)
```

Comment:
Should this link be: google-ad-manager#using-the-slot-and-fill-pattern ?

Reply:
Yes, sorry Michael. It should be `google-ad-manager.md#using-the-slot-and-fill-pattern`.